src/main/java/com/mentalhealthapi/mentalhealthapi/MentalHealthController.java

```diff
@@ -56,21 +58,17 @@ public ResponseEntity fetchBlogs(@PathVariable("name") String name) {
      */
     @PostMapping(value = "/disorders/{name}/blog", consumes = "application/json", produces = "application/json")
     public Blog createBlog(@PathVariable("name") String name, @RequestBody Blog blog) {
-        // we're going to need a user id passed here as well. for now, let's not worry about it
-
-        // for not, just return created blog as a stub
```

Comment:
removed unnecessary comments

Reply:
Good idea.  Avoid having conversation in source code comments.  Better to have them in GitHub pull request.